src/Import/NoFoundation.hs

```diff
@@ -1,12 +1,9 @@
 module Import.NoFoundation
-    ( module Import.NoFoundation
-    , module Import
+    ( module Import
     ) where
 
 import ClassyPrelude.Yesod   as Import
 import Settings              as Import
 import Yesod.Core.Types      as Import (loggerSet)
 import Yesod.Default.Config2 as Import
 import Control.Category      as Import ((>>>), (<<<))
-
-type One = Succ Zero
```

Comment:
Is this because you're building with a different set of dependencies than what is specified in stack.yaml?